src/scripts/db_store.js

```diff
@@ -0,0 +1,27 @@
+import browserID from "browser-id"
+import { getGraphqlHost } from "scripts/url"
+
+export const store_in_db = (customObject, ...args) => {
```

Comment:
```suggestion
export const uploadStatistics = (customObject, ...args) => {
```

Also: let's rename the file to something similar like `upload_statics.js`